test/unit/message_test.rb

```diff
@@ -41,16 +35,16 @@ class MessageTest < ActiveSupport::TestCase
     assert_equal "team", appreciation.team
   end
 
-  test "visible scope" do
+  test "index scope" do
     appreciation = FactoryBot.create(:appreciation)
     assert_equal [appreciation], Appreciation.all
-    assert_equal [appreciation], Appreciation.visible
+    assert_equal [appreciation], Appreciation.index
 
     appreciation.hide!
-    assert_equal Time.zone.now.to_s, appreciation.hidden_at.to_s
+    assert_equal Time.now.to_s, appreciation.hidden_at.to_s
```

Comment:
[pronto-rubocop warning] Rails/TimeZone: Do not use `Time.now` without zone. Use one of `Time.zone.now`, `Time.current`, `Time.now.in_time_zone`, `Time.now.utc`, `Time.now.getlocal`, `Time.now.xmlschema`, `Time.now.iso8601`, `Time.now.jisx0301`, `Time.now.rfc3339`, `Time.now.httpdate`, `Time.now.to_i`, `Time.now.to_f` instead.